script.js

```diff
@@ -1,54 +1,115 @@
-let bookArr = [];
+const booky = { id: new Date().getTime() };
```

Comment:
- [ ] Great job refactoring your code 👍 however according to the requirements you should you're not allowed to use objects or arrays outside of a class as described in the below point:
- Use JavaScript `classes` instead of `objects` and `arrays`.
  - You can use objects and arrays, but do it *inside* classes.